src/hl/ready.rs

```diff
@@ -146,7 +146,8 @@ where
         mut self,
         data: &[u8],
         send_time: SendTime,
-        config: &Config,
+        continuation: TxContinuation,
```

Comment:
I like that, but the number of arguments of this function is getting out of control.

Maybe wait until we get #21 's simplifications to the `send` function in?